tests/unit/test_functioncalls.py

```diff
@@ -0,0 +1,73 @@
+import numpy as np
+from unittest import TestCase
+import micro_manager
+
+
+class MicroSimulation:
+    def __init__(self):
+        self.very_important_value = 0
+
+    def initialize(self):
+        pass
+
+    def solve(self, macro_data, dt):
+        assert macro_data["macro-scalar-data"] == 1
+        assert macro_data["macro-vector-data"].tolist() == [0, 1, 2]
+        return {"micro-scalar-data": macro_data["macro-scalar-data"] + 1,
+                "micro-vector-data": macro_data["macro-vector-data"] + 1}
+
+
+class TestFunctioncalls(TestCase):
+
+    def test_micromanager_initialize(self):
+        manager = micro_manager.MicroManager('test_unit.json')
+        self.assertTrue(True)
```

Comment:
What are we testing here? Is it the constructor of the MicroManager class? I would add descriptions to all the tests to shortly explain what is being tested.

Reply:
Yes it tests the constructor, do you think it would also make sense to check values here?

Reply:
Yes lets check some values which are set inside the constructor rather than only checking if the constructor runs through. For example, any of the internal variables that are set based on what is provided in the configuration.